### ISSUE TITLE
Documentation: Missing dependencies for qemu in README

### ISSUE THREAD

**Issue by quambene** [wimpysworld/quickemu#89]
Add relevant dependencies for qemu installation in README, e.g.:

Qemu dependencies:

- SDL or GTK
- Spice
- VirtFS

Configure [qemu](https://www.qemu.org/download/) with 

``` bash
./configure --enable-sdl --enable-spice --enable-virtfs
```

or

``` bash
./configure --enable-gtk --enable-spice --enable-virtfs
```

**Comment by poleguy** [comment]
I agree, how to deal with QEMU is not obvious. These scripts promise to make this much easier, but the barrier to entry might mean that novices can't actually get quickemu running...

For example, I can't get this to run on ubuntu 20.04. The readme says there is a backport for QEMU 6.0.0 in the ppa, but there are no install instructions, and I can't get "quickemu --vm ..."  to run even after attempting to install QEMU. All I get is this error:

ERROR! Qemu 6.0.0 or newer is required, detected 4.2.1.

I tried uninstalling and reinstalling QEMU, but to no avail.

**Comment by flexiondotorg** [comment]
> For example, I can't get this to run on ubuntu 20.04. The readme says there is a backport for QEMU 6.0.0 in the ppa, but there are no install instructions, and I can't get "quickemu --vm ..." to run even after attempting to install QEMU. All I get is this error:
> 
> ERROR! Qemu 6.0.0 or newer is required, detected 4.2.1.

The installation instruction for Ubuntu are here:

  * https://github.com/wimpysworld/quickemu#ubuntu
